tools/depcheck/pkg/cmd/flags.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+)
+
+type GraphOptions struct {
+	Entrypoints []string
+	RepoRoot    string
+
+	Excludes []string
+	Filters  []string
+}
+
+type GraphFlags struct {
+	Filter  string
+	Exclude string
+
+	RepoRoot    string
+	Entrypoints []string
+}
+
+func (o *GraphFlags) ToOptions(out, errout io.Writer) (*GraphOptions, error) {
+	opts := &GraphOptions{
+		Entrypoints: o.Entrypoints,
+		RepoRoot:    o.RepoRoot,
+	}
+
+	if len(o.Exclude) > 0 {
+		f, err := ioutil.ReadFile(o.Exclude)
+		if err != nil {
+			return nil, err
+		}
+
+		err = json.Unmarshal(f, &opts.Excludes)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if len(o.Filter) > 0 {
+		f, err := ioutil.ReadFile(o.Filter)
+		if err != nil {
+			return nil, err
+		}
+
+		err = json.Unmarshal(f, &opts.Filters)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return opts, nil
+}
+
+func (o *GraphFlags) AddFlags(cmd *cobra.Command) {
```

Comment:
it's our command.  Add `--openshift` that builds the lists for us in `ToOptions` for openshift.  It's good to be king.